core/src/test/java/apoc/convert/ConvertTest.java

```diff
@@ -28,6 +29,11 @@ public class ConvertTest {
     @ClassRule
     public static DbmsRule db = new ImpermanentDbmsRule();
 
+    @After
```

Comment:
should this be AfterClass instead as we don't create/start the db per test method. and doing it per method would explode the test runtime.